src/core/net_db.h

```diff
@@ -55,6 +55,17 @@
 namespace i2p {
 namespace data {
 
+/// @enum NetDbDuration
+/// @brief Constants defining different refresh intervals
+///   for various NetDb operations
+enum struct NetDbDuration : const std::size_t {
```

Comment:
From what I see below, we won't need `std::size_t` when `std::uint16_t` will suffice.